site/_posts/2020-05-27-CSI-integration.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "How to use CSI Volume Snapshotting with Velero"
+excerpt: In the Velero 1.4 release, we introduced support for CSI snapshotting v1beta1 APIs. This post provides step-by-step instructions on setting up a CSI environment in Azure, installing Velero 1.4 with the velero-plugin-for-csi, and a demo of this feature in action.
+author_name: Ashish Amarnath
+image: /img/posts/csi-announce-blog.jpg
+# Tag should match author to drive author pages
+tags: ['Velero Team', 'Ashish Amarnath']
+---
+
+In the recent [1.4 release of Velero](https://github.com/vmware-tanzu/velero/releases), we announced a new feature of supporting CSI snapshotting using the [Kubernetes CSI Snapshot Beta APIs](https://kubernetes.io/docs/concepts/storage/volume-snapshots/).
+With this capability of CSI volume snapshotting, Velero can now support any volume provider that has a CSI driver with snapshotting capability, without requiring a Velero-specific volume snapshotter plugin to be available.
+
+This post has the necessary instructions for you to start using this feature.
+
+## Getting Started
+
+Using the CSI volume snapshotting features in Velero involves the following steps.
+
+1. Set up a CSI environment with a driver supporting the Kubernetes CSI snapshot beta APIs.
+1. Install Velero with CSI snapshotting feature enabled.
+1. Deploy `csi-app`: a pod using a CSI backed PVC that we will backup and restore.
+1. Use Velero to backup and restore the `csi-app`.
+
+## Set up a CSI environment.
+
+As the [Kubernetes CSI Snapshot Beta API](https://kubernetes.io/docs/concepts/storage/volume-snapshots/) is available starting from Kubernetes `1.17`, your will need to be running Kubernetes `1.17` or later.
+
+This post uses an AKS cluster running Kubernetes `1.17`, with Azure disk CSI driver as an example.
+
+Following instructions to install the Azure disk CSI driver from [here](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/install-csi-driver-master.md) run the below command
```

Comment:
Can change "Following instructions... " to "Follow [these instructions](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/install-csi-driver-master.md) to install the Azure Disk CSI driver, then run the command below:"

Reply:
running that script is part of the instructions. So going to leave this as-is